clean_ceph_vols.py

```diff
@@ -0,0 +1,141 @@
+#!/usr/bin/env python
+"""
+Utility to cleanup orphaned volumes.
+
+"""
+from gevent import monkey, sleep
+
+monkey.patch_all()
+import sys
+from datetime import datetime
+
+import yaml
+from docopt import docopt
+from libcloud.common.exceptions import BaseHTTPError
+from libcloud.common.types import LibcloudError
+from libcloud.compute.providers import get_driver
+from libcloud.compute.types import Provider
+
+from ceph.parallel import parallel
+from utility.retry import retry
+
+doc = """
+Utility to cleanup orphaned volumes.
+
+ Usage:
+  clean_ceph_vols.py [--osp-cred <file>]
+
+Options:
+  --osp-cred <file>                 openstack credentials as separate file [default: osp/osp-cred-ci-2.yaml]
+"""
+
+
+def get_openstack_driver(yaml):
+    OpenStack = get_driver(Provider.OPENSTACK)
+    glbs = yaml.get("globals")
+    os_cred = glbs.get("openstack-credentials")
+    username = os_cred["username"]
+    password = os_cred["password"]
+    auth_url = os_cred["auth-url"]
+    auth_version = os_cred["auth-version"]
+    tenant_name = os_cred["tenant-name"]
+    service_region = os_cred["service-region"]
+    domain_name = os_cred["domain"]
+    tenant_domain_id = os_cred["tenant-domain-id"]
+    driver = OpenStack(
+        username,
+        password,
+        ex_force_auth_url=auth_url,
+        ex_force_auth_version=auth_version,
+        ex_tenant_name=tenant_name,
+        ex_force_service_region=service_region,
+        ex_domain_name=domain_name,
+        ex_tenant_domain_id=tenant_domain_id,
+    )
+    return driver
+
+
+def volume_cleanup(volume, driver):
+    print("Removing volume", volume.name)
```

Comment:
Could we add docstring's for the method?

Reply:
Updated doc string